guides/gatherers.md

```diff
@@ -14,32 +14,49 @@ Facts Gathering process in a nutshell
+| [`fstab@v1`](#fstabv1)                       | [trento-project/agent/../gatherers/fstab.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/fstab.go)                     |
+| [`groups@v1`](#groupsv1)                     | [trento-project/agent/../gatherers/groups.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/groups.go)                   |
+| [`hosts@v1`](#hostsv1)                       | [trento-project/agent/../gatherers/hostsfile.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/hostsfile.go)             |
+| [`package_version@v1`](#package_versionv1)   | [trento-project/agent/../gatherers/packageversion.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/packageversion.go)   |
+| [`passwd@v1`](#passwdv1)                     | [trento-project/agent/../gatherers/passwd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/passwd.go)                   |
+| [`sapcontrol@v1`](#sapcontrolv1)             | [trento-project/agent/../gatherers/sapcontrol.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapcontrol.go)           |
+| [`saphostctrl@v1`](#saphostctrlv1)           | [trento-project/agent/../gatherers/saphostctrl.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/saphostctrl.go)         |
+| [`sap_profiles@v1`](#sap_profilesv1)         | [trento-project/agent/../gatherers/sapprofiles.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapprofiles.go)         |
+| [`saptune@v1`](#saptunev1)                   | [trento-project/agent/../gatherers/saptune.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/saptune.go)                 |
+| [`sbd_config@v1`](#sbd_configv1)             | [trento-project/agent/../gatherers/sbd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbd.go)                         |
+| [`sbd_dump@v1`](#sbd_dumpv1)                 | [trento-project/agent/../gatherers/sbddump.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbddump.go)                 |
+| [`sysctl@v1`](#sysctlv1)                     | [trento-project/agent/../gatherers/sysctl.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sysctl.go)                   |
+| [`systemd@v1`](#systemdv1)                   | [trento-project/agent/../gatherers/systemd.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/systemd.go)                 |
+| [`verify_password@v1`](#verify_passwordv1)   | [trento-project/agent/../gatherers/verifypassword.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/verifypassword.go)   |
+
+### cibadmin@v1
```

Comment:
Unfortunately, this breaks the redirecting in `hex` docs.
We will need to add the `span` trick, as I did for systemd@v2.
Other option is to only do the change in the index, and leave the `v1` version title without the version suffix